manage.py

```diff
@@ -105,5 +105,11 @@ def resetadminpassword(username, new_password):
     reset_admin_password(username, new_password)
 
 
+@manager.command
+def version():
+    """Returns the version of PostMaster"""
+    return '1.1.0.0-beta'
```

Comment:
Version should be a `__version__` variable in __init__.py that is imported into manager.py. Normally it's in setup.py but this is a flask application. config.py wouldn't be good either since that will be the user edited file that won't get updates from us.